frontend/src/metabase/components/EntityItem.jsx

```diff
@@ -22,6 +22,139 @@ const EntityItemWrapper = Flex.extend`
   }
 `;
 
+function getPinnedBackground(model) {
+  return model === "dashboard"
+    ? color("accent4")
+    : lighten(color("accent4"), 0.28);
+}
+
+function getPinnedForeground(model) {
+  return model === "dashboard" ? color("white") : color("accent4");
+}
+
+function getBackground(model) {
+  return model === "dashboard" ? color("brand") : color("brand-light");
+}
+
+function getForeground(model) {
+  return model === "dashboard" ? color("white") : color("brand");
+}
+
+function EntityItemIcon({
+  item,
+  variant,
+  iconName,
+  pinned,
+  selectable,
+  selected,
+  onToggleSelected,
+  ...props
+}) {
+  const iconSize = variant === "small" ? 12 : 18;
+  const handleClick = e => {
+    e.preventDefault();
+    onToggleSelected();
+  };
+
+  return (
+    <IconWrapper
```

Comment:
`IconWrapper` isn't very accessible as a button since it's just a div. I wonder if we can't add a new `IconButtonWrapper` or something since it seems that's what we typically use it for. Otherwise, we need to add `tabIndex={0}` and a `role` and `onMouseDown` handling to this.

Reply:
Thank you for keeping an eye on a11y. Added `IconButtonWrapper` [here](https://github.com/metabase/metabase/pull/16357/commits/0548f7cd1091edc4572a54abc17887d7e617e409), please have a look